docs/MD_SEARCH.rst

```diff
@@ -0,0 +1,263 @@
+      <Name>bucketname</Name>
+      <Prefix/>
+      <Marker/>
+      <MaxKeys>1000</MaxKeys>
+      <IsTruncated>false</IsTruncated>
+      <Contents>
+          <Key>objectKey</Key>
+          <LastModified>2018-04-19T18:31:49.426Z</LastModified>
+          <ETag>&quot;d41d8cd98f00b204e9800998ecf8427e&quot;</ETag>
+          <Size>0</Size>
+          <Owner>
+              <ID>79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2be</ID>
+              <DisplayName>Bart</DisplayName>
+          </Owner>
+          <StorageClass>STANDARD</StorageClass>
+      </Contents>
+      <Contents>
+          ...
+      </Contents>
+  </ListBucketResult>
+
+Performing Metadata Searches with Zenko
+---------------------------------------
+
+You can perform metadata searches by:
+
++ Using the :code:`search_bucket` tool in the
+  `Scality/S3 <https://github.com/scality/S3>`_ GitHub repository.
++ Creating a signed HTTP request to Zenko in your preferred programming
+  language.
+
+Using the S3 Tool
++++++++++++++++++
+
+After cloning the `Scality/S3 <https://github.com/scality/S3>`_ GitHub repository
+and installing the necessary dependencies, run the following command in the S3
+project’s root directory to access the search tool:
+
+.. code::
+
+  node bin/search_bucket
+
+This generates the following output:
+
+.. code::
+
+    Usage: search_bucket [options]
+
+    Options:
+
+      -V, --version                 output the version number
+      -a, --access-key <accessKey>  Access key id
+      -k, --secret-key <secretKey>  Secret access key
+      -b, --bucket <bucket>         Name of the bucket
+      -q, --query <query>           Search query
+      -h, --host <host>             Host of the server
+      -p, --port <port>             Port of the server
+      -s                            --ssl
+      -v, --verbose
+      -h, --help                    output usage information
+
+In the following examples, Zenko Server is accessible on endpoint
+:code:`http://127.0.0.1:8000` and contains the bucket :code:`zenkobucket`.
+
+.. code::
+
+    # search for objects with metadata "blue"
+    node bin/search_bucket -a accessKey1 -k verySecretKey1 -b zenkobucket \
+        -q "x-amz-meta-color=blue" -h 127.0.0.1 -p 8000
+
+    # search for objects tagged with "type=color"
+    node bin/search_bucket -a accessKey1 -k verySecretKey1 -b zenkobucket \
+        -q "tags.type=color" -h 127.0.0.1 -p 8000
+
+Coding Examples
++++++++++++++++
+
+Search requests can be also performed by making HTTP requests authenticated
+with one of the AWS Signature schemes: version 2 or version 4. \
+For more about authentication scheme, see:
+
+* https://docs.aws.amazon.com/general/latest/gr/signature-version-2.html
+* http://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html
+* http://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html
+
+You can also view examples for making requests with Auth V4 in various
+languages `here <../../../examples>`__.
```

Comment:
link leads to 404

Reply:
We have another follow up PR taking care of the leftovers.